packages/bodiless-components/src/Chameleon/withChameleonButton.tsx

```diff
@@ -37,13 +37,19 @@ const useToggleButtonMenuOption = () => {
 
 const useSwapButtonMenuOption = (formTitle: string = 'Choose a component') => {
   const { selectableComponents, activeComponent, setActiveComponent } = useChameleonContext();
-  const renderForm = () => {
+  const renderForm = ({ formApi }: any) => {
     const {
       ComponentFormLabel,
       ComponentFormRadioGroup,
       ComponentFormRadio,
       ComponentFormTitle,
     } = useMenuOptionUI();
+
```

Comment:
We shouldn't have to do this on effect. Informed should set it based on the initial values (confirmed [here](https://codesandbox.io/s/great-lamport-c5m70?file=/src/App.js)].  So we just need to make sure that we specify a default value and pass it through correctly.

Reply:
Updated